packages/contracts/hardhat-zksync.config.ts

```diff
@@ -102,7 +104,7 @@ const config: HardhatUserConfig = {
       url: 'http://127.0.0.1:8011',
       ethNetwork: 'http://127.0.0.1:8545',
       zksync: true,
-      deploy: ['./deploy/new'],
+      deploy: ['./deploy'],
```

Comment:
This differs from the change on OSx's counterpart. Is this intended?

Reply:
yes